### DISCUSSION TITLE
Ricedroidoss sync error

### DISCUSSION THREAD

**Discussion by ghost** [ROM-builders/temporary#17264] [General]
@Apon77 sir

https://cirrus-ci.com/task/6719912442331136?logs=sync#L438

What is the issue here?
I have built some ROMs 
But now a days I couldn't build any ROM becoz sync error everytimes

Please solve it as soon as possible
Thanks

**Comment by Lafactorial** [comment]
https://cirrus-ci.com/task/4540512925908992 and same here.
